connectors/github/element-templates/github-connector.json

```diff
@@ -656,6 +685,66 @@
         ]
       }
     },
+    {
+      "label": "Git reference",
+      "description": "Branch or tag name for the workflow",
+      "group": "input",
+      "type": "String",
+      "feel": "optional",
+      "value": "",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "gitReference"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "operationType",
+        "oneOf": [
+          "createWorkflowDispatchEvent"
+        ]
+      }
+    },
+    {
+      "label": "Workflow ID",
+      "description": "ID of the Github Actions workflow",
+      "group": "configuration",
+      "type": "String",
+      "feel": "optional",
+      "value": "",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "workflow_id"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "operationType",
+        "oneOf": [
+          "createWorkflowDispatchEvent"
+        ]
+      }
+    },
+    {
+      "label": "Workflow inputs",
+      "description": "Key-value pairs in a single JSON object",
```

Comment:
**Note for reviewers:** [API docs](https://docs.github.com/en/rest/actions/workflows?apiVersion=2022-11-28#create-a-workflow-dispatch-event) specify a 10 property maximum.

It appears these kinds of business logic limits usually aren't mentioned or enforced in element templates, but I wanted to flag here just in case.